src/utils/createTodo.ts

```diff
@@ -0,0 +1,14 @@
+function generateId(): number {
+  return Math.floor(Math.random() * 900000) + 100000;
+}
```

Comment:
I don't think you need to generate ID. After you send request to add a new todo, server returns this newly added todo with a generated ID.

Reply:
Thanks for catching the bug and mentioning the issue with the ID.